### ISSUE TITLE
Version 0.26.0 breaks webpack build: Can't resolve 'path' in .../monaco-editor/esm/vs/language/json/_deps/minimatch

### ISSUE THREAD

**Issue by akphi** [microsoft/monaco-editor#2578]
### Issue

We are using `monaco-editor` and `monaco-editor-webpack-plugin` in our [codebase](https://github.com/finos/legend-studio). When we upgrade from `monaco-editor@0.25.2` to `0.26.0` and we got a failure when building the app in webpack:

```
ModuleNotFoundError: Module not found: Error: Can't resolve 'path' in '/Users/<username>/projects/legend/studio/node_modules/monaco-editor/esm/vs/language/json/_deps/minimatch'

BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.

If you want to include a polyfill, you need to:
	- add a fallback 'resolve.fallback: { "path": require.resolve("path-browserify") }'
	- install 'path-browserify'
If you don't want to include a polyfill, you can use an empty module like this:
	resolve.fallback: { "path": false }
```

I looked at the file `monaco-editor/esm/vs/language/json/_deps/minimatch` and I see this:

```js
var path = { sep: '/' }
try {
  path = require('path')
} catch (er) {}
```

I'm afraid Webpack might look past the `try .. catch` block and yell about `path` there. Right now my workaround is as suggested in the error message, that is to add `resolve.fallback: { "path": false }`. I just wonder if this is something you would fix or would you recommend me to look for the answer from the `webpack` side instead?

### Environment

**monaco-editor version:** 0.26.0
**Browser:** Chrome Version 91.0.4472.114 (Official Build) (x86_64)
**OS:** Mac OSX 11.4
**Webpack:** 5.44.0
**monaco-editor-webpack-plugin:** 4.1.0

**Comment by troy351** [comment]
Also at the official site https://microsoft.github.io/monaco-editor/, change language to `json` shows error in console
![image](https://user-images.githubusercontent.com/8372385/125887436-aa2fe86a-499d-4cd6-86d1-17a87e39c4ef.png)

**Comment by lsaccoz** [comment]
> Also at the official site https://microsoft.github.io/monaco-editor/, change language to `json` shows error in console
> ![image](https://user-images.githubusercontent.com/8372385/125887436-aa2fe86a-499d-4cd6-86d1-17a87e39c4ef.png)

Been pulling my hair out over this issue in my app sine this afternoon. Seems like it's a general issue then.

**Comment by troy351** [comment]
Seems caused by new added `minimatch`  https://github.com/microsoft/monaco-json/commit/4c5bfeb135aaa196ba3c18c78c95f5d4f9ab54da
@aeschli any ideas?

**Comment by hediet** [comment]
Thanks for reaching out! I had a minimatch related error when publishing monaco-json, but @aeschli fixed it.
Seems like there is something missing.

**Comment by hediet** [comment]
We temporarily reverted monaco-json back to 3.5.0 and published 0.26.1.
